openpaas-james/mailbox/opendistro-tests/src/test/resources/eml/bodyMakeTikaToFail.eml

```diff
@@ -0,0 +1,1272 @@
+Date: Mon, 30 Jan 2017 11:51:35 +0100
```

Comment:
I don't care about behaviour when tika fails, it is not related to OpenDistro and it is a lot of LoC.

Please drop this file and remove related failing tests...